modules/ROOT/pages/_partials/afm-ref-schedule-interval.adoc

```diff
@@ -0,0 +1,41 @@
+[[scheduling-limits]]
+= Limits for Schedule Intervals by Location
+
+The minimum interval for schedules that run in private locations is always five minutes.
+
+The minimum interval for schedules that run in public locations depends on the number of schedule entitlements, or API monitor entitlements, that you have. If you have more than five, the limit is five minutes. Otherwise, it is fifteen minutes.
```

Comment:
I would remove the last sentence since the table shows this information.

Reply:
I'd like to leave this as deliberate redundancy, since it summarizes the information without someone having the cognitive load of analyzing the table. Also, I was asked to add that by Eng sometime ago, so want to honor that request.